common.js

```diff
@@ -11,6 +11,15 @@ export const translate = (key, replacements = []) => {
     return translation;
 }
 
+export const ariaLiveAnnouncementRegions = () => {
+    return `
+        <div id="srAnnouncerContainer" class="sr-only">
+            <div id="ariaLiveQuestionAnnouncer" aria-live="polite"></div>
```

Comment:
what other settings are there? :D

Reply:
ASSERTIVE!! lol. Joking aside, assertive takes immediate control and talks over everything else. Polite waits in the announcement queue.